package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "comment-related-issues",
-  "version": "0.0.7",
```

Comment:
Just go with `0.0.7` and use this command https://classic.yarnpkg.com/lang/en/docs/cli/version/

Reply:
wow
✅ bfee44206afc7f597fbda53edaafad4061135b39